docs/gen_ref_pages.py

```diff
@@ -23,9 +23,24 @@
         continue
 
     nav[parts] = doc_path.as_posix()
-    with mkdocs_gen_files.open(full_doc_path, "w") as fd:
-        ident = ".".join(parts)
-        fd.write(f"::: {ident}")
+    if full_doc_path.name != 'index.md':
+        with mkdocs_gen_files.open(full_doc_path, "w") as fd:
+            ident = ".".join(parts)
+            fd.write(f"::: {ident}")
+    else:
+        this_module_path = Path("src") / path.parent.name
+        module_index = ""
+        for module_scripts in sorted(this_module_path.rglob("*.py")):
+            if "__init__" in module_scripts.name:
+                continue
+            module_index += f"* [{module_scripts.name.replace('.py', '')}]" \
+                            f"({module_scripts.name.replace('.py', '.md')})\n"
+
+        with mkdocs_gen_files.open(full_doc_path, "w") as fd:
+            fd.write(module_index)
```

Comment:
what's with this new version? what does it do that the other doesn't? would like a comment explaining that.

Reply:
(so that I know whether to use this myself at some point!)

Reply:
I'll add a comment but it basically solves the problem of having an empty "Neurostatslib" page in the docs. Instead of that you'll have an index page with the list of modules

Reply:
oh cool, I was wondering the way to do this